65black/StackOlite
# #160199502 Get A Single Question

#### What does this PR do?
This PR consumes the `/api/v1/questions/:questionId` endpoint from [this repo](https://github.com/65black/StackOverflow-lite).

#### How should this be manually tested?
 - Clone this repo and open index.html with Chrome.
 - Navigate to the recent questions page.
 - Click on any question to view it.
 - You should be redirected to a page with the clicked question's details and answers. 

#### Screenshots
![question](https://user-images.githubusercontent.com/36507972/45905823-b4042400-bde9-11e8-8b56-7cafb9468ff6.JPG)



#### What are the relevant pivotal tracker stories?
#160199502
